### ISSUE TITLE
Canvas shifted up when not focused while loading (Wire)

### ISSUE THREAD

**Issue by MakisH** [saenzramiro/rambox#976]
### Steps to reproduce

1. Add Wire on Rambox
2. Quit Rambox and run it again
3. Don't focus the Wire tab, wait until you expect it to have loaded
4 Focus the Wire tab: everything is slighly shifted up.

Workaround:
Refresh the tab while having it focused and everything comes back into place.

### Expected behavior

All the page elements should shown, no blank gaps should appear.

### Actual behavior

The upper elements are cropped and there is a white gap at the end.

![wire-rambox-shifted](https://user-images.githubusercontent.com/4943683/27858754-b6b53746-6176-11e7-9f39-0cbe2a95bbac.png)

I haven't noticed this with Facebook Messenger, Gmail or Google Calendar.
I was not able to reproduce this on Firefox.

### ENVIRONMENT

Rambox: 0.5.10
Platform: linux (x64)
Electron: 1.7.4
Chromium: 58.0.3029.110
Node: 7.9.0

Wire: 2017-06-28

Update: just to confirm, this still holds in November 27, 2017 (Rambox 0.5.13)

**Comment by alien2003** [comment]
the same happens to me on Ubuntu Gnome 16.04

**Comment by saenzramiro** [comment]
I think this is an issue of Wire related with responsiveness. Because the first time you switch to a service once you opened Rambox, the webview size is around 400x200px and then we auto-size to fit the entire space.

**Comment by MakisH** [comment]
As a workaround, I also noticed that Wire responds correctly when I resize the Rambox window.

I tried to reproduce the same issue in Firefox and Chromium, but I could not find a way to. From Wire they also cannot find a relevance to their implementation (see https://github.com/wireapp/wire-webapp/issues/1616).

But let me ask, out of curiosity, why does the webview size need to be constant in the begining and then resize, rather than first set it to the current window's size and then load the websites? I hope I am not asking something very stupid here... :-/
